api/v1beta1/designatebackendbind9_types.go

```diff
@@ -78,6 +78,11 @@ type DesignateBackendbind9SpecBase struct {
 	// +kubebuilder:validation:Optional
 	// StorageRequest
 	StorageRequest string `json:"storageRequest"`
+
+	// +kubebuilder:default="internalapi"
```

Comment:
In retrospect, I don't think this is a good default. It may not be possible to to have a default really. What are your thoughts on leaving it empty  and only creating the services if a pool is provided?